src/Exception/HandlingFailureException.php

```diff
@@ -1,16 +0,0 @@
-<?php
-
-/*
- * This file is part of the Manala package.
- *
- * (c) Manala <contact@manala.io>
- *
- * For the full copyright and license information, please refer to the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Manala\Exception;
-
-class HandlingFailureException extends \RuntimeException
```

Comment:
This one is to keep (used by `Diff` and should probably be added to the `@throws` docblock section of [`Manala\Handler\Handler::handle()`](https://github.com/manala/manala/blob/master/src/Handler/Handler.php#L30))